.github/workflows/test.yml

```diff
@@ -38,7 +42,7 @@ jobs:
         run: make test
 
       - name: Run helper tests and sonar analysis
-        run: make test-docker SONAR_TOKEN=${{ secrets.SONAR_TOKEN }}
+        run: make test-docker SONAR_TOKEN=${{ secrets.SONAR_TOKEN }} LANG=${{ matrix.language }}
```

Comment:
Can we consolidate the `LANG` and `LIBRARIES` env vars into just one and call it `LANGUAGES`? This would simplify the logic.